main.css

```diff
@@ -1,64 +1,178 @@
-main {
+.cv {
   max-width: 1024px;
   margin: 0 auto;
+  padding: 0;
   font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
-  font-size: 20px;
+  font-size: 24px;
   border: solid;
-  background-color: darkblue;
+  background-color: rgb(10, 20, 90);
 }
-header {
+.head-part {
   display: flex;
-  flex-direction: row;
+  padding: 10px;
   flex-wrap: wrap;
   justify-content: space-around;
-  align-items: stretch;
-  background-color: silver;
-  color: darkblue;
-  border-bottom: solid;
-  margin: 5px;
-  padding: 5px;
-}
-h1 {
-  position: relative;
-  right: 15px;
-  text-align: center;
+  align-content: flex-start;
+  background-color: rgb(190, 190, 190);
+  color: rgb(10, 20, 90);
+  margin: 0;
 }
-.personalinfo {
-  color: black;
-  position: relative;
-  text-align: left;
-  margin: 20px;
-  left: 35px;
+.main-header {
+  font-size: 40px;
+  margin: 0;
+}
+.personal-info {
+  color: rgb(0, 0, 0);
   text-decoration: none;
 }
-h2 {
+.objective {
   text-align: center;
-  padding-top: 100px;
-  text-decoration: underline;
+  position: relative;
+  top: 50px;
+  font-size: 40px;
+  margin: 0;
+}
+.photo {
+  width: 400px;
+  height: 320px;
+  margin: 0;
 }
-div {
-  display: inline-block;
+.my-info {
   border: solid;
-  background-color: silver;
+  background-color: rgb(190, 190, 190);
   width: 470px;
   text-align: justify;
-  margin: 2px;
-  padding: 4px;
+  margin: 8px;
+  padding: 10px;
   border-radius: 20px;
 }
-section {
+.main-info {
   display: flex;
-  flex-direction: row;
   flex-wrap: wrap;
   justify-content: space-around;
 }
-h3 {
-  font-size: 26px;
+.info-header {
+  font-size: 30px;
   text-align: center;
-  text-decoration: underline;
-  color: darkblue;
+  color: rgb(10, 20, 90);
+  font-style: italic;
 }
 a[href] {
-  color: black;
+  color: rgb(0, 0, 0);
   text-decoration: none;
 }
+
+@media screen and (max-width: 1062px) {
+  .my-info {
+    width: 460px;
+  }
+}
+@media screen and (max-width: 1042px) {
+  .my-info {
+    width: 450px;
+  }
+}
+@media screen and (max-width: 1022px) {
+  .my-info {
+    width: 420px;
+  }
+}
+@media screen and (max-width: 962px) {
+  .cv {
+    font-size: 22px;
+  }
+  .main-header {
+    font-size: 32px;
+  }
+  .photo {
+    position: relative;
+    left: 30px;
+    width: 370px;
+    height: 300px;
+  }
+  .objective {
+    font-size: 32px;
+  }
+  .my-info {
+    width: 400px;
+  }
+  .info-header {
+    font-size: 30px;
+  }
+}
+@media screen and (max-width: 875px) {
+  .cv {
+    font-size: 20px;
+  }
+  .main-header {
+    font-size: 28px;
+  }
+  .photo {
+    position: relative;
+    left: 30px;
+    width: 370px;
+    height: 300px;
```

Comment:
All these styles already added for `@media screen and (max-width: 962px)`. That means that they will apply for all screens less then `962px`. As `875px` less then `962px` you don't need to repeat them.
Also check all the styles below there is no unnecessary repetitions